Makefile

```
@@ -3,6 +3,12 @@
 all:
 	@echo "Project successfully compiled"
 
+run-server:
+	bash .evergreen/run-orchestration.sh
```

```suggestion
	.evergreen/run-orchestration.sh
```
Prefer to use the script's shebang instead (+ ensure the script has executable permissions).

Also applies to `stop-orchestration.sh` below.